src/builder.ts

```diff
@@ -11,13 +11,13 @@ export class Builder {
 
   private projectDir = new PathHelper().getProjectDirectory();
 
-  public build() {
+  public build(): Promise<void> {
     console.log(chalk.yellow(chalk.italic('Building...')));
     return this.buildWithWebpack();
   }
 
   private buildWithWebpack() {
-    return new Promise((resolve, reject) => {
+    return new Promise<void>((resolve, reject) => {
```

Comment:
I guess you could as well define it in the method signature as well:

```ts
private buildWithWebpack(): Promise<void> {
```

Reply:
I think that's more elegant! For now I'd like to leave this function as it is, though, because it will change again in the next PR... 😉